wremnants/datasets/datagroups.py

```diff
@@ -297,9 +325,12 @@ def histNameCombine(procName, baseName, syst, channel):
         return name
 
 class datagroups2016(datagroups):
-    def __init__(self, infile, combine=False, pseudodata_pdfset = None, applySelection=True
+    def __init__(self, infile, combine=False, pseudodata_pdfset = None, applySelection=True,
+                 excludeProcGroup=None, filterProcGroup=None
     ):
-        self.datasets = {x.name : x for x in datasets2016.getDatasets()}
+        self.datasets = {x.name : x for x in datasets2016.getDatasets(filt=filterProcGroup, excludeGroup=excludeProcGroup)}
+        self.datasetsKeys = self.datasets.keys()
```

Comment:
Is it really worth defining a variable for this?

Reply:
no, it was to save some calls to .keys() elsewhere too, but now I dropped it